Framework/CCDBSupport/src/CCDBHelpers.cxx

```diff
@@ -240,6 +250,7 @@ auto populateCacheWith(std::shared_ptr<CCDBFetcherHelper> const& helper,
       helper->mapURL2UUID[path].lastCheckedTF = timingInfo.tfCounter;
       if (etag.empty()) {
         helper->mapURL2UUID[path].etag = headers["ETag"]; // update uuid
+        helper->mapURL2UUID[path].cacheValidUntil = headers["Cache-Valid-Until"].empty() ? 0 : std::stoul(headers["Cache-Valid-Until"]);
```

Comment:
It appears  that this `"Cache-Valid-Until"` is provided not at the 1st query (with empty etag) and at the 1st query with valid etag (actually at the 2nd one, see my email to @costing ). 
So, there is no point in its caching here, instead should be cached as done here https://github.com/AliceO2Group/AliceO2/blob/956b26710723080d559b30835eb15cadaba0d503/Framework/CCDBSupport/src/CCDBHelpers.cxx#L265 and also here https://github.com/AliceO2Group/AliceO2/blob/956b26710723080d559b30835eb15cadaba0d503/Framework/CCDBSupport/src/CCDBHelpers.cxx#L275 as 
```
else {
  helper->mapURL2UUID[path].cacheValidUntil = headers["Cache-Valid-Until"].empty() ? 0 : std::stoul(headers["Cache-Valid-Until"]);
}
```

Reply:
@shahor02 updated.